apiserver/facades/client/modelmanager/modelmanager.go

```diff
@@ -86,7 +86,6 @@ type ModelManagerAPI struct {
 	toolsFinder common.ToolsFinder
 
 	// Broker/Provider management.
-	getBroker      newCaasBrokerFunc
```

Comment:
hi tom,

I think this line could be removed as well right?

type newCaasBrokerFunc func(_ context.Context, args environs.OpenParams, _ environs.CredentialInvalidator) (caas.Broker, error)

Reply:
The mock broker as well :)

Reply:
Done + a few more I found :|